### DISCUSSION TITLE
Form Fields

### DISCUSSION THREAD

**Discussion by anishsarkar** [filamentphp/filament#821] [Help] [answered]
Hi, 

How do I show some fields in Edit Resource but turn them off in Create Resource. I see that there is a conditionally hiding field components in Form Section, but I am not sure how to use them. 

Any pointers? 

Thanks in advance.

**Comment by molcsab** [comment, marked as answer]
Hi,
This is very simple to do:
```php
->hidden(fn ($livewire) => !$livewire instanceof CreateRecord)
```

**Reply by anishsarkar** [reply]
Thanks! I guess it should work for ViewRecord also similarly. Anyway, going to test it.
